docs/topics/multiplatform-mobile/multiplatform-mobile-create-first-app.md

```diff
@@ -102,33 +111,6 @@ If you want to run your application on another simulated device, you can add a n
 >
 {type="note"}
 
-## Run tests
-
-You can run tests to check that the shared code works correctly on both platforms. Of course, you can also write and run tests to check the 
-platform-specific code.
-
-### Run tests on iOS
-    
-1. Open the file `iosTest.kt` in `shared/src/iosTest/kotlin/com.example.kmmapplication.shared`.  
-    Directories with `Test` in their name contain tests.  
-    This file includes a sample test for iOS.  
-    
-    ![iOS test Kotlin file](ios-test-kt.png)
-   
-2. Click the **Run** icon in the gutter next to the test.  
-
-Tests run on a simulator without UI. Congratulations! The test has passed – see test results in the console.
-
-![iOS test result](ios-test-result.png){width=300}
-
-### Run tests on Android
-
-For Android, follow a procedure that is very similar to the one for running tests on iOS.
-
-1. Open the file `androidTest.kt` in `shared/src/androidTest/kotlin/com.example.kmmapplication.shared`.
-
-2. Click the **Run** gutter icon next to the test. 
-
 ## Update your application
 
 1. Open the file `Greeting.kt` in `shared/src/commonMain/kotlin/com.example.kmmapplication.shared`.  
```

Comment:
The autogenerated package name for me is com.samples.myapplication not com.example.kmmapplication.shared